flixel/FlxSprite.hx

```diff
@@ -934,15 +934,16 @@ class FlxSprite extends FlxObject
 			}
 			#end
 			
-			var doFlipX = flipX != _frame.flipX;
-			var doFlipY = flipY != _frame.flipY;
+			var doFlipX:Bool = checkFlipX();
+			var doFlipY:Bool = checkFlipY();
+			
 			if (!doFlipX && !doFlipY && _frame.type == FlxFrameType.REGULAR)
 			{
 				framePixels = _frame.paint(framePixels, _flashPointZero, false, true);
 			}
 			else
 			{
-				framePixels = _frame.paintRotatedAndFlipped(framePixels, _flashPointZero, FlxFrameAngle.ANGLE_0, flipX, flipY, false, true);
+				framePixels = _frame.paintRotatedAndFlipped(framePixels, _flashPointZero, FlxFrameAngle.ANGLE_0, checkFlipX(), checkFlipY(), false, true);
```

Comment:
Why call the check functions again here when the results are locally cached a few lines earlier in `doFlipX` / `Y`?